src/components/NewsletterForm/Captcha.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { CaptchaDiv } from './NewsletterForm.styled';
+import type { Captcha as CaptchaType } from './NewsletterForm.types';
+
+export const Captcha: CaptchaType = () => (
+  <CaptchaDiv
+    className="ml-form-recaptcha ml-validate-required"
+    style={{ float: 'left' }}
+  >
+    <div
+      className="g-recaptcha"
+      data-sitekey="6Lf1KHQUAAAAAFNKEX1hdSWCS3mRMv4FlFaNslaD"
```

Comment:
tego nie mozesz trzymać w kodzie - to musi byc jako zmienna i trzymana w `.env` a potem na pipeline

Reply:
zrobione